test/unit/lib/plugins/aws/package/compile/events/stream.test.js

```diff
@@ -1215,6 +1223,10 @@ describe('AwsCompileStreamEvents', () => {
           awsCompileStreamEvents.serverless.service.provider.compiledCloudFormationTemplate
             .Resources.FirstEventSourceMappingKinesisBar.Properties.ParallelizationFactor
         ).to.equal(undefined);
+        expect(
```

Comment:
It would be better to implement these tests using `runServerless` as described in: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md

In this case it should exetend these tests: https://github.com/serverless/serverless/blob/master/test/unit/lib/plugins/aws/package/compile/events/stream.test.js#L1585

Reply:
Should I remove these ones? Also, I couldn't find a reference to `runServerless` in the markdown

Reply:
Yes, please remove the added assertions to old tests and confirm it's working correctly via `runServerless` :+1: Sorry - I was supposed to provide this link: https://github.com/serverless/serverless/blob/master/test/README.md